### PR TITLE
Add beta label to cody web chat

### PR DESCRIPTION
**Pull request by thenamankumar**
Add beta label to cody web chat.

image

## Test plan

- visit /cody

### CODE REVIEW

**Review by abeatrix — APPROVED** [review]

**Review by efritz — COMMENTED** [review]
👍

**Review by efritz — COMMENTED** [review]
👍

**Comment by sourcegraph-buildkite** [comment]
## Bundle size report 📦

| Initial size | Total size | Async size | Modules |
| --- | --- | --- | --- |
| 0.00% (0.00 kb) | 0.00% (+0.15 kb) | 0.00% (+0.15 kb) | 0.00% (0) |

Look at the [Statoscope report](https://console.cloud.google.com/storage/browser/_details/sourcegraph_reports/statoscope-reports/naman/cody-web-beta/compare-report.html;tab=live_object) for a full comparison between the commits c329802 and ce84251 or [learn more](https://docs.sourcegraph.com/dev/how-to/testing#bundlesize).


<details>
<summary>Open explanation</summary>

- `Initial size` is the size of the initial bundle (the one that is loaded when you open the page)
- `Total size` is the size of the initial bundle + all the async loaded chunks
- `Async size` is the size of all the async loaded chunks
- `Modules` is the number of modules in the initial bundle
</details>

File: client/web/src/cody/chat/CodyChatPage.tsx [hunk 1]
```diff
@@ -78,7 +79,14 @@ export const CodyChatPage: React.FunctionComponent<CodyChatPageProps> = ({ authe
                 className="mb-3"
             >
                 <PageHeader.Heading as="h2" styleAs="h1">
-                    <PageHeader.Breadcrumb icon={CodyColorIcon}> Cody Chat</PageHeader.Breadcrumb>
+                    <PageHeader.Breadcrumb icon={CodyColorIcon}>
+                        <div className="d-inline-flex align-items-center">
+                            Cody Chat
+                            <Badge variant="info" className="ml-2">
```
Comment (eseliger) [inline client/web/src/cody/chat/CodyChatPage.tsx:85]: There is the ProductStatusBadge for this 🙂

**Pull request merged**